aicsimageio/readers/tiff_reader.py

```diff
@@ -175,53 +193,93 @@ def _create_dask_array(self) -> da.Array:
+                # outer-most with the chunks as long as the dimension is size 1
+                blocked_dim_order = non_chunk_dim_order + chunk_dim_order
+                blocked_shape = tuple(non_chunk_shape) + ((1,) * len(chunk_shape))
 
                 # Make ndarray for lazy arrays to fill
-                lazy_arrays = np.ndarray(operating_shape, dtype=object)
+                lazy_arrays = np.ndarray(blocked_shape, dtype=object)
                 for plane_index, (np_index, _) in enumerate(
                     np.ndenumerate(lazy_arrays)
                 ):
+                    # All dimensions get their normal index except for chunk dims
+                    # which get filled with "full" slices
+                    indicies_with_slices = np_index[: len(non_chunk_shape)] + (
+                        (slice(None, None, None),) * len(chunk_shape)
+                    )
+
                     # Fill the numpy array with the delayed arrays
                     lazy_arrays[np_index] = da.from_delayed(
                         delayed(TiffReader._get_image_data)(
                             fs=self.fs,
                             path=self.path,
                             scene=self.current_scene_index,
-                            index=plane_index,
+                            indicies=indicies_with_slices,
                         ),
-                        shape=sample.shape,
-                        dtype=sample.dtype,
+                        shape=chunk_shape,
+                        dtype=selected_scene.dtype,
                     )
 
                 # Convert the numpy array of lazy readers into a dask array
                 image_data = da.block(lazy_arrays.tolist())
 
+                # Because we have set certain dimensions to be chunked and others not
+                # we will need to transpose back to original dimension ordering
+                # Example, if the original dimension ordering was "TZYX" and we
+                # chunked by "T", "Y", and "X"
+                # we created an array with dimensions ordering "ZTYX"
+                transpose_indices = []
+                transpose_required = False
+                for i, d in enumerate(selected_scene_dims):
+                    new_index = blocked_dim_order.index(d)
+                    if new_index != i:
+                        transpose_required = True
+                        transpose_indices.append(new_index)
+                    else:
+                        transpose_indices.append(i)
+
+                # Only run if the transpose is actually required
+                if transpose_required:
+                    image_data = da.transpose(image_data, tuple(transpose_indices))
+
                 return image_data
 
     def _read_delayed(self) -> xr.DataArray:
         """
-        Construct the delayed xarray DataArray object for the image.
+        Creates a delayed dask array for the file.
```

Comment:
do you call `.compute()` on the returned `xr.DataArray` ?

Reply:
I think this happened as a part of my attempt at merging `master`. Will revert.